src/main/java/frc/robot/subsystems/drive/SwerveModule.java

```diff
@@ -165,9 +191,12 @@ public void burnFlashSparks() {
   public SwerveModuleState getState() {
     // Apply chassis angular offset to the encoder position to get the position
     // relative to the chassis.
+    // return new SwerveModuleState(
+    //     drivingTalon.getVelocity().getValue(),
+    //     new Rotation2d(turningAbsoluteEncoder.getPosition() - chassisAngularOffsetRadians));
     return new SwerveModuleState(
         drivingTalon.getVelocity().getValue(),
-        new Rotation2d(turningEncoder.getPosition() - chassisAngularOffsetRadians));
+        new Rotation2d(turningRelativeEncoder.getPosition() - chassisAngularOffsetRadians));
```

Comment:
If we zero the module correctly then I don't think we need to add/subtract anything anymore. Also in other places

Reply:
Done